app/Libraries/Search/BeatmapsetSearch.php

```diff
@@ -33,7 +33,19 @@ public function __construct(?BeatmapsetSearchParams $params = null)
      */
     public function getQuery()
     {
-        static $partialMatchFields = ['artist', 'artist.*', 'artist_unicode', 'creator', 'title', 'title.raw', 'title.*', 'title_unicode', 'tags^0.5'];
+        static $partialMatchFields = [
+            'artist',
+            'artist.*',
+            'artist_unicode',
+            'artist_unicode.*',
+            'creator',
+            'title',
+            'title.raw',
```

Comment:
not sure if useful considering there's also `title.*`

Reply:
Should be fine to remove 🤔